ezmap-poc.sln

```diff
@@ -2,17 +2,27 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{41F11E7F-D94F-49D7-A341-F31DCCAFA179}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "backend", "backend", "{D02B54F0-A63E-4C1F-AA84-8D3E8FDAFB81}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EzMap.Domain", "EzMap.Domain\EzMap.Domain.csproj", "{633AC4D5-D664-495A-8083-41E444C0D92C}"
```

Comment:
Path should reflect solution structure and vice versa
i.e. physical path should be `src\EzMap.Domain\EzMap.Domain.csproj`
Same for EzMap.Api